data/brands/amenity/fuel.json

```diff
@@ -3050,14 +3050,22 @@
       }
     },
     {
-      "displayName": "NIS",
+      "displayName": "НИС Петрол",
       "id": "nis-a971aa",
       "locationSet": {"include": ["rs"]},
       "tags": {
         "amenity": "fuel",
-        "brand": "NIS",
+        "brand": "НИС Петрол",
         "brand:wikidata": "Q1279721",
-        "name": "NIS"
+        "name": "НИС Петрол",
+        "name:sr": "НИС Петрол",
+        "name:sr-Latn": "NIS Petrol",
+        "contact:email": "office@nis.rs",
```

Comment:
Besides, the same reason as https://github.com/osmlab/name-suggestion-index/pull/7073#discussion_r979248251, we don't collect such detailed contact data, and we can't assume that every branch amenity share this. The only thing they should share is the canonical brand features itself.